native/cocos/bindings/jswrapper/v8/MissingSymbols.h

```diff
@@ -28,21 +28,4 @@
 #include <cstddef>
 #include <cstring>
 
-#if SCRIPT_ENGINE_TYPE == SCRIPT_ENGINE_V8
-    #if CC_PLATFORM == CC_PLATFORM_ANDROID
-        #if __ANDROID_API__ < 21
-extern "C" {
-int localSigemptyset(void *set) {
-    if (set == NULL) { // NOLINT(modernize-use-nullptr)
-        errno = EINVAL;
-        return -1;
-    }
-    memset(set, 0, sizeof(unsigned long)); // NOLINT(google-runtime-int)
-    return 0;
-}
-
-int sigemptyset(void *set) __attribute__((weak, alias("localSigemptyset")));
-}
-        #endif
-    #endif
-#endif // #if SCRIPT_ENGINE_TYPE == SCRIPT_ENGINE_V8
```

Comment:
v8 11.6 doesn't need this hack.
It's probably we used the wrong building arguments while compiling v8 9.1.